### ISSUE TITLE
bin/start-dev is not working.

### ISSUE THREAD

**Issue by leeduckgo** [ArweaveTeam/arweave#330]
===> Compiling c_src/b64fast.c
===> Linking /home/ubuntu/arweave/_build/default/lib/b64fast/priv/b64fast.so
===> "/home/ubuntu/arweave/_build/default/lib/prometheus_cowboy/ebin/prometheus_cowboy.app" is missing description entry
CMake Error: The source directory "/home/ubuntu/arweave/apps/arweave/lib/RandomX" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
===> Hook for compile failed!

**Comment by rosmcmahon** [comment]
You need to git clone using the --recursive flag.

https://github.com/arweaveteam/arweave#requirements

**Comment by leeduckgo** [comment]
> You need to git clone using the --recursive flag.
> 
> https://github.com/arweaveteam/arweave#requirements

thx a lot, but it's still no working. Is it possible to put start-dev( I thought it's to run a local test node but I am not sure) in the release package?

**Comment by leeduckgo** [comment]
> Member

thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?

**Comment by ldmberman** [comment]
> thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?

yep, https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992

**Comment by leeduckgo** [comment]
> > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> 
> yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)

how can I do this? is there any guide for dev mode?

**Comment by ahhxfeng** [comment]
have you finish this ? build local dev mode ? i have the same question

**Comment by leeduckgo** [comment]
> > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > 
> > 
> > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> 
> how can I do this? is there any guide for dev mode?

Not yet.

**Comment by ahhxfeng** [comment]
> 

> > > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > > 
> > > 
> > > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> > 
> > 
> > how can I do this? is there any guide for dev mode?
> 
> Not yet.



> > > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > > 
> > > 
> > > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> > 
> > 
> > how can I do this? is there any guide for dev mode?
> 
> Not yet.
> lol, so it support to do this or not ?

**Comment by leeduckgo** [comment]
> > 
> 
> > > > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > > > 
> > > > 
> > > > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> > > 
> > > 
> > > how can I do this? is there any guide for dev mode?
> > 
> > 
> > Not yet.
> 
> > > > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > > > 
> > > > 
> > > > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> > > 
> > > 
> > > how can I do this? is there any guide for dev mode?
> > 
> > 
> > Not yet.
> > lol, so it support to do this or not ?

At last we we put this attempt on pending...

**Comment by ahhxfeng** [comment]
> > > 
> > 
> > 
> > > > > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > > > > 
> > > > > 
> > > > > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> > > > 
> > > > 
> > > > how can I do this? is there any guide for dev mode?
> > > 
> > > 
> > > Not yet.
> > 
> > 
> > > > > > thx, bro. so, is it possible to run a arweave local dev network by our-self(only 1 node is ok, just like ethereum local dev node)?
> > > > > 
> > > > > 
> > > > > yep, [#329 (comment)](https://github.com/ArweaveTeam/arweave/issues/329#issuecomment-1029445992)
> > > > 
> > > > 
> > > > how can I do this? is there any guide for dev mode?
> > > 
> > > 
> > > Not yet.
> > > lol, so it support to do this or not ?
> 
> At last we we put this attempt on pending...

lol....
